Button/Button.module.less

```diff
@@ -4,6 +4,16 @@
 @import "../styles/variables.less";
 @import "../styles/skin.less";
 
+@keyframes scale {
+	0%   { transform: scale(0); }
+	100% { transform: scale(1); }
+}
+@keyframes scale-bounce {
```

Comment:
This animation conflicts with the styling of buttons in the electro skin.

@Djspaceg - Do you want to address that?

Reply:
I'll look into this real quick.

Reply:
This will require a much more elaborate animation system, with animations that are specific to a skin, which isn't currently implemented. `@keyframe` sequences can't be wrapped by selectors, and therefore fall outside the realm of skin's reach. We need a system that provides a global cache of skin-name-prefixed animations which can reference skin variables to be able to support this and a proper animation bank.

FWIW, the following is what the keyframes will look like to properly implement this across all skins:
```
@keyframes scale-bounce {
	0%   { transform: scale(0); animation-timing-function: ease-out; }
	80%  { transform: scale(1.07); animation-timing-function: ease-in-out; }
	100% { transform: scale(1); }
}

@keyframes scale-bounce-bg {
	0%   { transform: @agate-button-bg-transform, scale(0); animation-timing-function: ease-out; }
	80%  { transform: @agate-button-bg-transform, scale(1.07); animation-timing-function: ease-in-out; }
	100% { transform: @agate-button-bg-transform, scale(1); }
}
```
I was working on a system to support this, but had to back-burner it for today's work priority.

Reply:
For clarity, you'd like to move ahead with this PR then?

Reply:
I think so. I don't have a better solution. I could include a note about what you noticed and why we "chose what we chose" for this case.

Reply:
This trail is sufficient I think.